docs/source/traits_user_manual/defining.rst

```diff
@@ -132,7 +132,7 @@ in the following table.
    pair: type; Unicode
 .. index:: Boolean type, Bool trait, CBool trait, Complex trait, CComplex trait
 .. index:: complex number type, Float trait, CFloat trait, Int trait, CInt trait
-.. index:: floating point number type, Long trait, CLong trait, Str trait
+.. index:: floating point number type, Str trait
```

Comment:
If we choose to deprecate (but nore remove in this PR) the `Long` and `CLong` traits, the entries should stay in the table and the default values should be updated.

Reply:
Let's get rid of the uncertainty: we'll remove rather than deprecate. In internal conversations there was no-one wanting to keep `Long` and `CLong`.